packages/api/tests/lambdas/test-sqs-message-consumer.js

```diff
@@ -362,13 +341,14 @@ test.serial('SQS message consumer queues correct number of workflows for rules m
   t.is(queueMessageStub.callCount, 1);
 
   t.teardown(async () => {
-    await cleanupRulesAndQueues(rules, [queue]);
+    await cleanupQueues([queue]);
   });
 });
 
 test.serial('processQueues archives messages from the ENABLED sqs rule only', async (t) => {
   const { stackName } = process.env;
   const { rules, queues } = await createRulesAndQueues();
+  t.context.fetchRulesStub.returns(rules.filter((rule) => rule.state === 'ENABLED'));
```

Comment:
This seems like bad test setup: isn't the point of the test to ensure that the logic for `processQueues` only acts on ENABLED rules even if enabled/disabled rules exist? By mocking `fetchRules` this way, you're creating a scenario where only enabled rules exist. It seems like the mocking should include disabled and enabled rules?

Reply:
🤔 I'm open to having it return both types, on the other hand it feels a little outdated because the call to `fetchRules` includes `state: 'ENABLED'`, as did the previous call to `queryRules`, so this is testing a case that should never actually happen as disabled rules should never be in the return. Thoughts?

Reply:
I could change the expectation to test that `state: 'ENABLED'` is passed to the fetchRules call?

Reply:
https://github.com/nasa/cumulus/pull/2296/commits/58441b880cb6f2084dac63113a0dddda10553183

Reply:
Talking it out: you're mocking `fetchRules`, which the sqsMessageConsumer calls with `state: ENABLED`. I guess my point was that this test is explicitly named to state that it only processes ENABLED rules. That logic is enforced by the call to `fetchRules`, so I guess it **shouldn't** be possible for it to ever return DISABLED rules. Even if it was, that would be a problem with `fetchRules` to be handled by its unit tests. 

It seems like this test is designed to prove that even if there are DISABLED/ENABLED rules, then only the latter are processed. With this way of mocking, my concern sort of evaporates, but I wonder if it's not worth avoiding the mock for this test and creating ENABLED/DISABLED rules? If not, does this test really have value?

Reply:
I think as refactored it has value in terms of verifying the correct params are passed to `fetchRules`. What do you think?

Reply:
That works for me